app/src/js/reducers/granules.js

```diff
@@ -159,6 +164,30 @@ export default function reducer (state = initialState, action) {
       set(state, ['removed', id, 'error'], action.error);
       break;
 
+    case BULK_GRANULE:
+    //   data = {
+    //     “createdAt”: 1574730504000,
+    //     “id”: “0eb8e809-8790-5409-1239-bcd9e8d28b8e”,
+    //     “updatedAt”: 1574730504762,
+    //     “status”: “RUNNING”,
+    //     “taskArn”: “arn:aws:ecs:us-east-1:111111111111:task/d481e76e-f5fc-9c1c-2411-fa13779b111a”
+    // }
+      // console.log('in case granule_BULK', data);
+      // console.log('id in reducer', requestId);
```

Comment:
Any reason to keep the code commented out above? Can we just remove it?

Reply:
ahh good catch, I will remove